hooks/pre/32-install_selinux_packages.rb

```diff
@@ -2,17 +2,12 @@
 if facts.dig(:os, :selinux, :enabled)
   packages = []
 
-  if el7?
-    packages << 'rh-postgresql12-postgresql-server' if local_postgresql?
-    packages << 'rh-redis5-redis' if local_redis?
-  end
-
   packages << 'foreman-selinux' if foreman_server?
   packages << 'katello-selinux' if katello_enabled?
   packages << 'candlepin-selinux' if katello_enabled?
   packages << 'pulpcore-selinux' if pulpcore_enabled?
 
-  if katello_enabled? && el8?
+  if katello_enabled?
```

Comment:
I think this entire hack is no longer needed now that we have proper DNF modules. That would allow us to get rid of the `el8?` definition too.